webapp/src/app/shared/widgets/character-widget/character-widget.component.scss

```diff
@@ -0,0 +1,12 @@
+@import "~@angular/material/theming";
+
+$config: mat-typography-config();
+
+.text {
+					
+	$font-type:		body-1;
+	font-family:	mat-font-family($config);
```

Comment:
after `:` you can use spaces, no need to align all values in the same column.

Reply:
Just went with default-input since the nested is not relevant to this pull request.